packages/story-editor/src/components/library/panes/shopping/shoppingPane.js

```diff
@@ -113,23 +112,48 @@ function ShoppingPane(props) {
     [getProducts, showSnackbar]
   );
 
+  const searchProductsWithTelemetry = useCallback(
+    (value = '', sortBy, sortOrder) => {
+      searchProducts((value = ''), sortBy, sortOrder);
```

Comment:
Think we need to get rid of the = '' here ya? otherwise we don't end up using the search term